Userland/Utilities/CMakeLists.txt

```diff
@@ -11,59 +11,54 @@ list(APPEND RECOMMENDED_TARGETS
     nc netstat notify ntpquery open pape passwd pls printf pro shot tar tt unzip zip
 )
 
-# FIXME: Support specifying component dependencies for utilities (e.g. WebSocket for telws)
+set(JAKT_TARGETS "")
+if (ENABLE_JAKT)
+    add_jakt_executable(hello-jakt hello-jakt.jakt)
```

Comment:
Why the rename?

Reply:
No reason now, previously I was looking at having it just use the filename for the executable name for a single file, changed it back.